### ISSUE TITLE
Text does not match code output

### ISSUE THREAD

**Issue by mersennez** [keras-team/keras-io#447]
The **conclusions** in https://keras.io/examples/structured_data/imbalanced_classification/
do not match output of training (?)

**Comment by sachinprasadhs** [comment]
Hi,

Thanks for reporting the issue.

Since these documents are rerun for each result, the outcome is not exactly guaranteed with respect to the previous version.

The result which you see is approximate value of the validation set, if you increse the number of epochs you will again see the difference in these numbers. 

You can interpret the result by seeing validation metrics as compared to the below epoch results.

Correctly identifying 66 of them as fraudulent ~ `val_tp`
Missing 9 fraudulent transactions ~ `val_fn`
At the cost of incorrectly flagging 441 legitimate transactions ~ `val_fp`

![image](https://github.com/keras-team/keras-io/assets/73069040/34ee6f9f-94a9-4aa2-b842-da9a481ff24e)
